services/blockchain-app-registry/shared/utils/downloadRepository.js

```diff
@@ -199,8 +199,8 @@ const buildEventPayload = async (allFilesModified) => {
 };
 
 const isMetadataFile = (filePath) => (
-	filePath !== undefined
-	&& (filePath.endsWith(FILENAME.APP_JSON) || filePath.endsWith(FILENAME.NATIVETOKENS_JSON))
+	!!(filePath
```

Comment:
What was the issue with `filePath !== undefined`?

Reply:
the function returns `undefined` instead of boolean value.